installer/app/src/views/azure-create-app-tutorial.js.jsx

```diff
@@ -5,163 +5,240 @@ import Sheet from './css/sheet';
+            </div>
+        );
+
+
+        tutorialSlides.push(
+            <div id="tutorial-07">
+                <img src={AssetPaths["azure-7.gif"]} style={{
+                        width: 699,
+                        height: 402
+                    }} />
+                <p>Click the "Delegated Permissions" dropdown and check "Access Azure Service Management". Then, save the configuration.</p>
+            </div>
+        );
+
+        tutorialSlides.push(
+            <div id="tutorial-08">
+                <img src={AssetPaths["azure-8.gif"]} style={{
+                        width: 699,
+                        height: 472
+                    }} />
+                <p>Click on the back arrow button to go back to the "APPLICATIONS" tab and click and the "ENDPOINTS" button at the bottom. Then, copy your OAuth 2.0 Token Endpoint into the input below.</p>
+            </div>
+        );
+
+        tutorialSlides.push(
+            <div id="tutorial-09">
+                <img src={AssetPaths["azure-done.png"]} style={{
+                        width: 699,
+                        height: 256
+                    }} />
+                <p>You now created an appliation able to control your Azure resources - one that this Flynn installer can use. You can connect this installer and your new app by clicking "Authenticate" below - lets install Flynn!</p>
+            </div>
+        );
+
+        intro = (
+            <div id="azure-tutorial-intro">
+                <p>To use the installer, you first need to create an Azure application able to control your resources on Flynns behalf.</p>
+                <button onClick={this.__handleAdvanceTutorialClick}>Walk me through it</button><br />
+                <button onClick={this.__skipTutorial}>Skip tutorial</button>
+            </div>
+        );
+
+        intro = (s.tutorialSlide || s.skipTutorial) ? null : intro;
+        tutorialSlide = (s.tutorialSlide) ? tutorialSlides[s.tutorialSlide] : null;
+
+        if (s.tutorialSlide || s.skipTutorial) {
+            pagination.prev = (s.skipTutorial && !s.tutorialSlide) ? <button id="azureTutPrev" onClick={this.__skipTutorial}>Back</button> : '';
+            pagination.prev = (s.tutorialSlide > 1) ? <button id="azureTutPrev" onClick={this.__handleAdvanceTutorialClick}>Back</button> : pagination.prev;
+            pagination.next = (s.tutorialSlide < 9) ? <button id="azureTutNext" onClick={this.__handleAdvanceTutorialClick}>Next</button> : '';
+            pagination.submit = (s.skipTutorial || s.tutorialSlide === 9) ? <button type="submit" style={{margin: "0 0 0 503px"}}>Authenticate</button> : '';
+        }
+
+        return (
+            <div id={this.state.styleEl.id}>
+                {intro}
+                {tutorialSlide}
+
+                <div id="azure-tutorial-inputs">
+                    <label for="redirectURI" style={inputStyles.redirectURI}>Redirect URI</label>
```

Comment:
`s/for/htmlFor/` (https://facebook.github.io/react/docs/tags-and-attributes.html)